apps/masterbots.ai/components/routes/browse/browse-chat-messages.tsx

```
@@ -13,69 +13,69 @@
+		if (threadId && !messages.length) {
+			const messages = await getMessages({ threadId: threadId })
+			setMessages(messages)
+		}
+	}, [threadId, messages.length])
 
-  // Effect to fetch messages when the thread ID changes
-  React.useEffect(() => {
-    fetchMessages()
-  }, [threadId])
+	// Effect to fetch messages when the thread ID changes
+	React.useEffect(() => {
```

Biome (1.9.4)

[error] This hook specifies more dependencies than necessary: threadId

Outer scope values aren't valid dependencies because mutating them doesn't re-render the component.


(lint/correctness/useExhaustiveDependencies)